prisma-client-js/README.md

```diff
@@ -357,14 +361,16 @@ model Post {
 The above schema results in a `where` argument with the following type signature:
 
 ```ts
-type where = {
-  id?: number
-  email?: string
-  category_title?: { category: string; title: string }
-}
+type where =
+  | number // based on @id or @@id
```

Comment:
What number could be passed in the case of a compound primary key, e.g.
```prisma	
model Post {	
	category String	
	title    String	
	@@id([category, title])	
}
```

Reply:
In the case of a compound key it would be `{ where: { category: "...", title: "..." } }`